python/src/lib.rs

```diff
@@ -143,7 +143,11 @@ fn glide(_py: Python, m: &PyModule) -> PyResult<()> {
             }
             Value::Double(double) => Ok(PyFloat::new(py, double).into_py(py)),
             Value::Boolean(boolean) => Ok(PyBool::new(py, boolean).into_py(py)),
-            Value::VerbatimString { format: _, text } => Ok(text.into_py(py)),
+            Value::VerbatimString { format: _, text } => {
+                // TODO create MATCH on the format
+                let data_bytes = PyBytes::new(py, text.as_bytes());
+                Ok(data_bytes.into_py(py))
```

Comment:
Why return bytes from `SimpleString` and `VerbatimString` and then downcast to strings?
These types never contain binary strings.

Reply:
Here is the definition of the verbatim string type:

> Verbatim string: a binary safe string that should be displayed to humans without any escaping or filtering. For instance the output of LATENCY DOCTOR in Redis.

Reply:
Sorry, I still don't understand why `VerbatimString` gets `String -> byte[] -> string` conversion.

Reply:
it's just String -> bytes, its not being converted to string again